### PR TITLE
Make module compatible with Decidim v0.27

### PR DESCRIPTION
**Pull request by tramuntanal**
- Make module compatible with Decidim 0.27
- Change uses of Rectify gem for Decidim's replacements
- Use Ruby 3.1.3, be compatible with Ruby 3.0.2

### CODE REVIEW

**Review by NicoleeSo13 — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 = Changelog
 
+== 0.2.0 MINOR (02/05/2023)
+- Make module compatibile with Decidim 0.27
```
Comment (laurajaime) [inline CHANGELOG.md]:
```suggestion
- Make module compatible with Decidim 0.27
```

**Review by laurajaime — COMMENTED** [review]
Great! only one typo :)

**Comment by tramuntanal** [comment]
Fixed @laurajaime

**Review by laurajaime — APPROVED** [review]
LGTM!

**Pull request merged**